include/curl/curl.h

```diff
@@ -3128,6 +3132,7 @@ typedef struct curl_version_info_data curl_version_info_data;
 #define CURL_VERSION_HSTS         (1<<28) /* HSTS is supported */
 #define CURL_VERSION_GSASL        (1<<29) /* libgsasl is supported */
 #define CURL_VERSION_THREADSAFE   (1<<30) /* libcurl API is thread-safe */
+#define CURL_VERSION_NGTCP2       (1<<31) /* libngtcp2 is supported */
```

Comment:
This is not really a "feature" for us. ngtcp2 is a library, HTTP3 is the feature.

Reply:
OK, I've removed this anyway. Thanks for the clarification.